src/registry/content/Content.scss

```diff
@@ -0,0 +1,76 @@
+@use '@/scss/bem';
+@use '@utrecht/components/link';
+@use 'microscope-sass/lib/typography';
```

Comment:
I'd expect a `@use '@utrecht/components/html-content;` rule too to load the CSS for that component, which enables us to specify and use design tokens for that.

Reply:
I have added this but there are no design tokens for this. It only provides styles for the `utrecht-html` class (which is used in the `HTMLContent` component).